core/src/main/java/io/aiven/klaw/service/AclControllerService.java

```diff
@@ -509,6 +512,66 @@ public ApiResponse deleteAclRequests(String req_no) throws KlawException {
     }
   }
 
+  public ApiResponse claimAcl(int aclId) throws KlawException {
+    log.info("claimAcl {}", aclId);
+
+    if (commonUtilsService.isNotAuthorizedUser(
+        getPrincipal(), PermissionType.REQUEST_CREATE_SUBSCRIPTIONS)) {
+      return ApiResponse.NOT_AUTHORIZED;
+    }
+
+    final String userName = getCurrentUserName();
+    int tenantId = commonUtilsService.getTenantId(userName);
+
+    // Get ACL
+    Optional<Acl> aclOp = manageDatabase.getHandleDbRequests().getAcl(aclId, tenantId);
+    if (aclOp.isEmpty()) {
+      return ApiResponse.notOk("Acl does not exist.");
+    }
+
+    if (manageDatabase
+        .getHandleDbRequests()
+        .existsAclRequest(
+            aclOp.get().getTopicname(),
+            RequestStatus.CREATED.value,
+            aclOp.get().getEnvironment(),
+            tenantId)) {
+      return ApiResponse.notOk("Claim already exists for this Acl.");
+    }
+
+    // Copy into ACL Request
+    AclRequests request = new AclRequests();
+    copyProperties(aclOp.get(), request);
+
+    // reset the request number
+    request.setReq_no(null);
+    // Store the original aclId in the other Params section
+    request.setOtherParams(String.valueOf(aclOp.get().getReq_no()));
```

Comment:
do we want to create any other fld specific to this ?

Reply:
Line 547 setting the Req_no to null has it create a new request number.
Although I think it would be nice to add an aclid column, and am happy to do that and not use the otherParams.
Will look at doing that now